src/witan/send/adroddiad/clerk/slides.clj

```diff
@@ -13,39 +13,60 @@
 (defn bulleted-list [seq-of-text]
   (reduce #(into %1 [[:li.text-3xl.mb-4.mt-4 %2]]) [:ul.list-disc] seq-of-text))
 
-(defn chart-box [chart]
-  "expects a map for a vega-lite chart"
+(defmulti left-box :left-box)
+
+(defmethod left-box ::was/chart [conf]
+  (clerk/vl
+   {::clerk/width :full}
+   (:chart conf)))
+
+(defmethod left-box ::was/text [conf]
+  (clerk/html
+   {::clerk/width :full}
+   [:div.text-1xl.max-w-screen-2xl.font-sans
+    (bulleted-list (:text conf))]))
```

Comment:
this is a bit I'm glad is a multimethod as I would probably do my text as paragraphs rather than a list

Reply:
would love to see someone else use it!